.travis.yml

```diff
@@ -24,7 +24,7 @@ install:
     - yarn install
     - bower install
 script:
-    - yarn test:cover
```

Comment:
We dont need test coverage right now. Our tests first off are basically nonexistant, and coverage isn't a priority rn.